src/app/chart/donut/donut-config.ts

```diff
@@ -0,0 +1,36 @@
+import { ChartConfig } from '../chart-config';
+/**
+ * A config containing properties for the sparkline chart
+ */
+export class DonutConfig extends ChartConfig {
+
+  /**
+   * An optional function to customize the text of the center label
+   */
+  centerLabelFn?: any;
```

Comment:
In general, we want to stay away from callback functions. Instead, this can be a simple centerLabel property.

Reply:
Changed to `centerLabel` property.